### DISCUSSION TITLE
Currently, the Dependabot configuration in our repository is set to check for Python dependencies in the root directory. I wish to update it to target the backend directory.

### DISCUSSION THREAD

**Discussion by KariukiAntony** [fastapi/full-stack-fastapi-template#1141] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
# The desired state
# python dependencies
- package-ecosystem: "pip"
  directory: "backend"
  schedule:
  interval: "daily"
  commit-message:
  prefix: ⬆
```

### problem
 Currently, the Dependabot configuration in our repository is set to check for Python dependencies in the root directory.
### Proposed Solution:
I propose updating the Dependabot configuration to target the backend directory specifically for python dependency updates.

### Operating System

Linux

### Operating System Details

ubuntu 20.04

### Python Version

3.8

### Additional Context

I welcome feedback and discussion from the maintainers and community members on this proposal.

**Comment by YuriiMotov** [comment]
Currently its:
```
  # Python uv
  - package-ecosystem: uv
    directory: /backend
    schedule:
      interval: daily
    commit-message:
      prefix: ⬆
```
